junit5/src/main/java/cz/xtf/junit5/extensions/SinceVersionCondition.java

```diff
@@ -31,9 +34,22 @@ public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext con
 	}
 
 	public static ConditionEvaluationResult resolve(SinceVersion sinceVersion) {
+		if (sinceVersion.name().equals("") == sinceVersion.dockerImageName().equals("")) {
+			throw new RuntimeException("Only one of 'name' and 'dockerImageName' can be presented in 'SkipFor' annotation.");
+		}
 		Image image = Image.resolve(sinceVersion.image());
-		Pattern name = Pattern.compile(sinceVersion.name());
-		if (name.matcher(image.getRepo()).matches()) {
+		Matcher matcher;
+
+		if (!sinceVersion.name().equals("")) {
+			matcher = Pattern.compile(sinceVersion.name()).matcher(image.getRepo());
+		} else {
+			DockerImageMetadata metadata = DockerImageMetadata.get(OpenShifts.master(), image);
+			String[] imageNameParts = metadata.labels().get("name").split("/");
```

Comment:
I think we should not take part after `/` in label name. In case that `name` is:
```"name": "jboss-eap-7/eap73-openjdk8-openshift-rhel7"``` 

then let's take the whole string. Otherwise I think it's confusing. Taking part after `/` had sense only in case where `name` and `dockerImageName` were the same.

Reply:
Agreed, the part before the `/` might be important (if we want to filter out tech preview images for example).

Reply:
The first part is repository. The second one is actual image name. I see your point, it might be usefull. With @mnovak1 proposal about changing the parameter name (`ImageMetadataLabelName`) it should be pretty straightforward.